### PR TITLE
fixes: minor, feature: use default thumbnail image rendition in content list management

### PR DESCRIPTION
**Pull request by tomaszrondio**

### CODE REVIEW

File: client/controllers/WebPublisherContentListsController.js [hunk 1]
```diff
@@ -42,6 +42,21 @@ export function WebPublisherContentListsController($scope, publisher, modal, $ti
...
         }
 
+        /**
+         * @ngdoc method
+         * @name WebPublisherContentListsController#getThumbnail
+         * @param {Object} article - article content object
+         * @description Sets the active view name to the given value
+         */
+        getThumbnail(article) {
+            let base = article.tenant.subdomain ? 'http://' + article.tenant.subdomain + '.' + article.tenant.domainName : 'http://' + article.tenant.domainName;
```
Comment (takeit) [inline client/controllers/WebPublisherContentListsController.js]: it should be without the protocol: `://`.  The protocol can be omitted to not throw any warnings  I guess.

File: client/directives/listArticles/ListArticlesDirective.js [hunk 2]
```diff
@@ -38,6 +38,21 @@ export function ListArticlesDirective(publisher) {
...
             };
 
+            /**
+             * @ngdoc method
+             * @name sdListArticles#getThumbnail
+             * @param {Object} article
+             * @returns {String}
+             * @description Returns template url dependent on type
+             */
+            scope.getThumbnail = function(article) {
```
Comment (takeit) [inline client/directives/listArticles/ListArticlesDirective.js]: this method is duplicated. I think you can use a service and put it there.

File: client/views/settings/website-management/partials/tenant.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="sd-grid-item-header">
     <h3 class="sd-grid-item-header__heading sd-overflow-ellipsis">
-        <a ng-href="{{site.outputChannel ? site.outputChannel.config.url : site.subdomain ? 'http://' + site.subdomain + '.' + site.domainName: 'http://' + site.domainName}}" flow="down">
+        <a target="_blank" ng-href="{{site.outputChannel ? site.outputChannel.config.url : site.subdomain ? 'http://' + site.subdomain + '.' + site.domainName: 'http://' + site.domainName}}" flow="down">
```
Comment (takeit) [inline client/views/settings/website-management/partials/tenant.html]: protocol should be omitted

Comment (tomaszrondio) [reply client/views/settings/website-management/partials/tenant.html]: it is an url

File: client/controllers/WebPublisherContentListsController.js (hunk 1 above)
Comment (tomaszrondio) [reply client/controllers/WebPublisherContentListsController.js]: true

File: client/directives/listArticles/ListArticlesDirective.js (hunk 2 above)
Comment (tomaszrondio) [reply client/directives/listArticles/ListArticlesDirective.js]: good point

File: client/views/settings/website-management/partials/tenant.html (hunk 3 above)
Comment (takeit) [reply client/views/settings/website-management/partials/tenant.html]: yep but still it should open website according to the current protocol

Comment (tomaszrondio) [reply client/views/settings/website-management/partials/tenant.html]: nope bcs website can handle http only where superdesk instance should use https by default.

File: client/services/index.js [hunk 4]
```diff
@@ -1,2 +1,3 @@
 export {PublisherFactory} from './PublisherFactory';
 export {PubAPIFactory} from './PubAPIFactory';
+export {PublisherHelpersFactory} from './PublisherHelpers';
```
Comment (ahilles107) [inline client/services/index.js]: @tomaszrondio - it shouldn't be `'./publisherHelpers';`? As file is named `client/services/publisherHelpers.js`.

**Pull request merged**